docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-2.1-applications-linux-containers-aks-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+
+![Context menu option to add Docker support to a existing project: Right click (on the project) > Add > Docker Support.](media/add-docker-support-to-project.png)
+
+**Figure 4-38**. Adding Docker support to existing project
+
+To complete adding Docker support, you have the choice of Windows or Linux, in this case select **Linux**, because AKS doesn’t support Windows Containers (as of late 2018).
+
+![Option dialog to select Target OS for Dockerfile.](media/select-linux-docker-support.png)
+
+**Figure 4-39**. Selecting Linux Containers.
+
+With these simple steps, you will have your ASP.NET Core 2.1 application running on a Linux Container.
+
+As you can see the, integration between Visual Studio 2017 and Docker is totally oriented to the developer’s productivity.
+
+Now you can run your application with the key **F5** or using the Play Button
+
+After running the project, you can list the images using the `docker images` command, and you should see the `mssampleapplication` image created with the automatic deploy of our project with Visual Studio 2017.
+
+```console
+# docker images
+```
+
+![Console output from the docker images command, shows a list with: Repository, Tag, Image Id, Created (date), and Size.](media/docker-images-command.png)
+
+**Figure 4-40**. View of docker images
+
+## Register the Solution in the Azure Container Registry
+
+Now, we have to upload the image to any Docker registry, like [Azure Container Registry (ACR)](https://azure.microsoft.com/services/container-registry/) or Docker Hub so the images can be deployed to the AKS cluster from that registry. In this case, we’re uploading the image to Azure Container Registry.
+
+### Create the image in Release mode
+
+We'll now create the image in **Release** Mode (ready for production) changing to Release as shown here and running the application as did before.
+
+![Toolbar option in VS to build in release mode.](media/select-release-mode.png)
+
+**Figure 4-41**. Selecting Release Mode
+
+If you execute the `docker image` command, you'll see both images created one for `debug` and the other for `release` mode.
+
+### Create a new Tag for the Image
+
+Each container image needs to be tagged with the `loginServer` name of the registry. This tag is used for routing when pushing container images to an image registry.
+
+You can view the `loginServer` name from the Azure portal, taking the information from the Azure Container Registry
+
+![Browser view of the Azure container registry name, on the top right.](media/loginServer-name.png)
+
+**Figure 4-42**. View of the name of the Registry
+
+Or by running the command
+
+```console
+az acr list --resource-group MSSampleResourceGroup --query "[].{acrLoginServer:loginServer}" --output table
+```
+
+![Console output from the above command.](media/az-cli-loginServer-name.png)
+
+**Figure 4-43**. Get the name of the registry using PowerShell
+
+In both cases, you'll obtain the name, in our example `mssampleacr.azurecr.io`
```

Comment:
Break the sentence after the name. Add comma after example.

Reply:
done.